src/editor.rs

```diff
@@ -24,6 +25,13 @@ pub enum Mode {
     Insert,
 }
 
+pub enum RefreshOrder {
+    CursorPos,
+    Lines(HashSet<u16>),
+    StatusBar,
+    ALLLines,
```

Comment:
```suggestion
    AllLines,
```
C'est un peu moins aggressif comme ça 😅.